js/src/components/modals/SEMrushCountrySelector.js

```diff
@@ -239,8 +335,9 @@ class SEMrushCountrySelector extends Component {
 						</select>
 						<button
 							className="yoast-button yoast-button--secondary"
-							onClick={ this.newRequest }
-						>Change Country</button>
+							onClick={ this.relatedKeyphrasesRequest }
+						>Change Country
+						</button>
 					</FieldGroup>
 				</div>
```

Comment:
Pre-existing: not sure why we're using a wrapping `<div>` element _and_ a Fragment.